### ISSUE TITLE
Reflections (mirrors) are confusing

### ISSUE THREAD

**Issue by slava110** [SamsTheNerd/HexGloop#29]
By reading the manual I've expected reflections to work this way:
I should be able to bind mirrored pedestal to my player using pattern. After binding item in the pedestal will be counted as item in my offhand (if there's no item in my offhand) during casting (for read/write iota, for placing blocks, etc.).
In reality:
- Place pattern only works inside casting circle
- Write/read iota patterns don't work at all on reflected items

After trying to understand how to work with mirrored pedestal I've checked out source code. And it looks like this block serves completely different purpose as spell circle component? Not specified in Patchouli docs.
You can put item with pattern inside (like focus) and it will execute this iota if I understand correctly.
While it's cool I think that mirrored pedestal should function a different way. It should probably bind item as offhand item automatically like player would do with reflection bind spell. And it should probably be documented in Patchouli docs. What do you think?

**Comment by SamsTheNerd** [comment]
yeah, I was definitely worried about how reflections were communicated, it does help a lot to see how other people read it though !

so the place block thing not grabbing it outside of circles is a bug, I just never tested it oops.

when i started making the mirror system I started with hand mirrors, which is why read/write didn't pass through, since you need those to interact with the mirror. Then i decided to keep that limitation to avoid the mirror being a long range iota transfer system. It should probably be better documented though.

The circle functionality is documented in the [Loci Section](https://hexgloop.hexxy.media/#greatwork/loci). It's supposed to be hinted at with the whole "i sense more magic here" line, but having it explicitly link to the loci page once you're enlightened might be a good idea, especially for the online docs.

**Comment by slava110** [comment]
> when i started making the mirror system I started with hand mirrors, which is why read/write didn't pass through, since you need those to interact with the mirror. Then i decided to keep that limitation to avoid the mirror being a long range iota transfer system. 

I don't see a problem with long-range iota transfer system?.. Why is it bad? For me it's logical: item is literally reflected to your hand. You should be able to read from it or write to it.
Haven't tried hand mirror yet, I'll give them a try later
> The circle functionality is documented in the [Loci Section](https://hexgloop.hexxy.media/#greatwork/loci). It's supposed to be hinted at with the whole "i sense more magic here" line, but having it explicitly link to the loci page once you're enlightened might be a good idea, especially for the online docs.

I see... So autobinding won't work for mirrored pedestal because they already have different function. Okay then.

**Comment by SamsTheNerd** [comment]
idk, i could probably be talked into it, I just feel like it under cuts other forms of iota transfer. I suppose it still has the tradeoff of requiring an off hand though.
